batter_percent_overlay:
  ros__parameters:
    input_topic_name: /main_battery
    input_message_type: sensor_msgs/msg/BatteryState
    output_topic_name: /battery_percentage_overlay
    output_message_type: wifi_viz/msg/MinMaxCurr
    field_mappings_str: "percentage:current"
    static_fields_str: "min=0.0,max=1.0"
